faunadb-android/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="com.faunadb.client">
+    <application/>
```

Comment:
This is just to satisfy android build tools. That file won't be part of the final artifact